### ISSUE TITLE
[ptfiles] Login failed: No form found on https://ptfiles.net/loginproc/ using form selector form

### ISSUE THREAD

**Issue by cwfryer** [Jackett/Jackett#12154]
### Do not delete the template.

**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed. Also check our [Troubleshooting](https://github.com/Jackett/Jackett/wiki/Troubleshooting) for steps to resolve common issues.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**: Linux Mint 20.2

**.Net Runtime**: Mono

**.Net Version**:

**Jackett Version**: 0.18.537

**Last Working Jackett Version**: 0.18.536(?)

**Are you using a proxy or VPN?** Yes

### Description

1) Add PTFiles tracker to Jacket.
2) Click on settings
3) Input username
4) Input password
5) Click okay

### Logged Error Messages

An error occurred while updating this indexer
Login failed: No form found on https://ptfiles.net/loginproc/ using form selector form#login

Full traceback:

(categories) arguments accordingly. E.g. ...&cat=2030,2040&q=big+buck+bunny will search for "big buck bunny" in the Movies/SD (2030) and Movies/HD (2040) categories (See the indexer configuration for available categories).
Jackett Configuration
  

Admin password: 
Blank to disable
 
Base path override: 
/jackett
Server port: 
9117
Blackhole directory: 
c:\torrents\
Proxy type: 
Disabled
Proxy URL: 
Proxy port: 
Proxy username: 
Proxy password: 
External access: 
Disable auto update: 
Update to pre-release: 
Enhanced logging: 
Cache enabled (recommended): 
Cache TTL (seconds): 
2100
Cache max results per indexer: 
1000
FlareSolverr API URL: 
Blank for default
OMDB API key: 
OMDB API Url: 
Blank for default
Jackett Version v0.18.537
×
Server Logs
Date	Level	Message
2021-08-11 21:27:34	Error	
Jackett.Common.ExceptionWithConfigData: Login failed: No form found on https://ptfiles.net/loginproc/ using form selector form#login
   at Jackett.Common.Indexers.CardigannIndexer.DoLogin() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 570
   at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 965
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 97
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61

**Comment by garfield69** [comment]
The indexer works for me so the issue appears to be at your end.

Turn on enhanced logging so we can see the HTML and debug messages and we may be able to find something useful.
see https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket

It is likely that you are being blocked by your ISP or something else that is preventing the site from returning the login form.

**Comment by cwfryer** [comment]
[full_logs.txt](https://github.com/Jackett/Jackett/files/6979370/full_logs.txt)

Full logs attached.
I can access the indexer directly through the URL without any issues on my ISP both with, and without my VPN.

**Comment by garfield69** [comment]
well something is interfering with the response, because the log you provided shows that instead of getting a 3142 byte HTML page in response to a GET for https://ptfiles.net/loginproc/ , Jackett was instead given 74 bytes, which unfortunately the logger flagged as BINARY so we do not get to see the content in the log :-( 

@ilike2burnthing Any ideas how to progress investigating this one?

**Comment by ilike2burnthing** [comment]
Other than testing without the VPN, I'm not sure.

**Comment by cwfryer** [comment]
I am marking the issue as closed because I got it working, although the steps are unconventional and may not work for everyone.

The indexer stopped working when I updated my password.

I updated my password because I couldn't remember it.

I am running jackett inside a docker container. I killed the container and re-pulled a fresh image. At this point I noticed PTFiles was added, but the password looked short. I dug around in the config file and found my old password.

I went and changed my password back to my old password and it works fine once again.

Maybe I had an escape character in my new password or something... Idk.

Thanks for the help @garfield69.
